source/extensions/omni.isaac.lab/omni/isaac/lab/managers/manager_term_cfg.py

```diff
@@ -200,21 +200,24 @@ class EventTermCfg(ManagerTermBaseCfg):
     """
 
     is_global_time: bool = False
-    """ Whether randomization should be tracked on a per-environment basis.
+    """Whether randomization should be tracked on a per-environment basis. Defaults to False.
 
-    If True, the same time for the interval is tracked for all the environments instead of
-    tracking the time per-environment.
+    If True, the same interval time is used for all the environment instances.
+    If False, the interval time is sampled independently for each environment instance
+    and the term is applied when the current time hits the interval time for that instance.
 
     Note:
         This is only used if the mode is ``"interval"``.
     """
 
     min_step_count_between_reset: int = 0
```

Comment:
@kellyguo11 @Dhoeller19 I didn't quite understand this config term. Why do we call it `min`? To me, min would mean that after these many steps it is called every time. But from the code, it really means that it is triggered after these many steps. Shouldn't it just be `step_count_between_reset`?

Reply:
Also not a huge fan that we have intervals in dt and this thing in steps. We should stick to one, no?

Reply:
@Mayankm96 sure let's rename it to `step_count_between_reset`.
We could also switch to seconds and switch the input argument to `apply` from `global_env_step_count` to `global_env_time`

Reply:
> user might think it will be applied after that exact number of steps

exact number of steps since the last time the event was called 🤔 

Maybe better name could be: `num_steps_since_last_reset_event_trigger` though the name becomes long.

Reply:
I leave this for future. I think this is a breaking change which isn't the intention of this particular MR.